packages/bot-web-ui/src/components/load-modal/local-footer.tsx

```diff
@@ -34,6 +38,8 @@ const LocalFooter = ({
                     loadFileFromLocal();
                     toggleLoadModal();
                     setPreviewOnPopup(false);
+                    setToastMessage('import');
+                    setShowToast(true);
```

Comment:
@farabi-deriv if  2 of these functions are from the` load store` and 3 of them are from the `dashboard store`, then why can't we optimize it? we don’t need to connect to the store and pass 3 functions to call all of them here. We can create a new one function in the `dashboard store`, call all 3 functions there and pass 1 function to the current component.
use here like this:
```
type TSetOpenSettings = {
 has_preview_popup: boolean;
 toast_message: string;
 has_show_toast: boolean;
};
setOpenSettings = ({ false, 'import', true }: TSetOpenSettings) ;
```
the same approach with `load's functions store`, or them you can leave, because at least them only 2, but 3 from one store no sense to pass

Reply:
Done ✅